health/micro-ui/web/micro-ui-internals/packages/modules/health-hrms/src/components/pageComponents/ReportingUserSearchTable.js

```
@@ -0,0 +1,115 @@
+    onSelect?.(user);
+  };
+
+  // Columns configuration
+  const columns = useMemo(
+    () => [
+      {
+        name: t("CORE_COMMON_NAME"),
+        selector: (row) => row?.name?.givenName || "-",
+        sortable: true,
+        cell: (row) => (
+          <div style={{ padding: "8px", cursor: "pointer" }} onClick={() => handleSelect(row)}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)